services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelperV2.groovy

```diff
@@ -142,11 +145,26 @@ class FlowHelperV2 {
         log.debug("Adding flow '${flow.flowId}'")
         def response = northboundV2.addFlow(flow)
 
-        Wrappers.wait(WAIT_OFFSET) { assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP }
+        def flowEntry = null
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP
+
+            flowEntry = db.getFlow(flow.flowId)
+            assert flowEntry
+        }
+        checkRulesOnSwitches(flowEntry, RULES_INSTALLATION_TIME, true)
```

Comment:
I thought we no longer need this wait since v2 api only changes status to Up after all rules are _actually_ installed and validated

Reply:
Agree. Removed.
We also need to replace such checks in the update and delete methods with corresponding tests to verify that rules are actually on switches after operations are completed.